synfig-core/src/modules/lyr_std/perspective.cpp

```diff
@@ -66,12 +66,12 @@ using namespace lyr_std;
 
 /* === G L O B A L S ======================================================= */
 
-SYNFIG_LAYER_INIT(Warp);
-SYNFIG_LAYER_SET_NAME(Warp,"warp");
-SYNFIG_LAYER_SET_LOCAL_NAME(Warp,N_("Warp"));
-SYNFIG_LAYER_SET_CATEGORY(Warp,N_("Distortions"));
-SYNFIG_LAYER_SET_VERSION(Warp,"0.2");
-SYNFIG_LAYER_SET_CVS_ID(Warp,"$Id$");
+SYNFIG_LAYER_INIT(Perspective);
+SYNFIG_LAYER_SET_NAME(Perspective,"perspective");
```

Comment:
To support old versions, i think you need to set name here back to "warp".
It is not visible in GUI, but used for file saving/reading.

So try to change it back:
```
SYNFIG_LAYER_SET_NAME(Perspective,"warp");
```
and check if this will be enough,

Reply:
Yeah it worked thanks.